techdocs/docs/authentication-flow.md

```diff
@@ -0,0 +1,5 @@
+# Authentication Flow
+
+Below is a flow diagram of how authentication works between the SSO packages. 
+
+![alt text](../../assets/sso_auth_flow.PNG)
```

Comment:
not sure if this path will work till pushing to dev site

Reply:
Might need to link to the image url but we can try this